nodejs/eks/tsconfig.json

```diff
@@ -16,6 +16,7 @@
         "allowSyntheticDefaultImports": true
     },
     "files": [
+        "ami.ts",
```

Comment:
Do you know if there is a reason we include individual files here instead of just `**/*.ts`?

Reply:
No I don't think there is any. I can clean it up separately. Now's a good time because we're touching those files anyways